user-guide/Advanced_Modules/EPM/Working_with_the_EPM_user_interface/EPM_Topology_app.md

```diff
@@ -0,0 +1,45 @@
+---
+uid: EPM_Topology_app
+---
+
+# Using the EPM Topology app
+
+From DataMiner Cube 10.4.0 [CU14]/10.5.0 [CU2]/10.5.5 onwards<!-- RN 42221 -->, if your system has been correctly [configured to display the EPM Topology app](xref:Topology_app_configuration), you can access this app using the *Topology* button in the Cube sidebar. In earlier DataMiner versions, this feature is available in preview if the *CPEIntegration* [soft-launch option](xref:SoftLaunchOptions) is enabled.
+
+![Topology app](~/user-guide/images/EPMIntegration_Topology_app.png)<br>
+*Topology app in DataMiner Cube 10.5.5*
+
+## Navigating to a topology level
+
+In the filter pane on the left, you can select the front-end EPM Manager (in case more than one is available), select the topology chain, and drill down to any of the topology levels in that chain. You can then click the ">" button next to that level to open the corresponding page.
+
+![Topology app navigation](~/user-guide/images/EPMIntegration_Open_card.png)<br>
+*Topology app in DataMiner Cube 10.5.5*
+
+The page for the topology level will display the data configured for this level in the EPM Manager protocol. When you navigate between (docked) EPM cards, the filter selection will be updated to match the displayed card.
+
+If a topology level has child items, you can find these on the *Below this [EPM level name]* page.
+
+![Topology app child items](~/user-guide/images/EPMIntegration_Below_this_page.png)<br>
+*Topology app in DataMiner Cube 10.5.5*
+
+On the *Topology* page, you can find an overview of the topology for the selected level:
```

Comment:
double clicking on a block in the topology overview will navigate to that level and update the filter sections